.travis.yml

```diff
@@ -45,7 +45,7 @@ addons:
     - libmount-dev
 before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install gtk+3 cairo atk; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install libffi gtk+3 cairo atk; fi
```

Comment:
libffi is a dependency of glib, so this is only a workaround for brew being broken. Can you add a comment about that here?

Reply:
Sure!